doc/source/whatsnew/v1.3.0.rst

```diff
@@ -117,6 +117,9 @@ We provided some focused development on :class:`.Styler`, including altering met
 to accept more universal CSS language for arguments, such as ``'color:red;'`` instead of
 ``[('color', 'red')]`` (:issue:`39564`). This is also added to the built-in methods
 to allow custom CSS highlighting instead of default background coloring (:issue:`40242`).
+Enhancements to other built-in methods include extending the :meth:`.Styler.background_gradient`
+method to shade elements based on a given gradient map and not be restricted only to
+values in the DataFrame (:issue:`39930`).
```

Comment:
emm, just one nitpick, is `39930` an issue?

Reply:
no thats the PR actually, which references the two issues: closes #22727 closes #28901. Is there a sphinx directive for PRs as opposed to issues? I wasn't previously aware of this.

Reply:
nah, i don't think there is sphinx for prs, if so, would suggest to use ```(:issue:`22727`, :issue:`28901`)```, instead of ```(:issue:`39930`)```. Since usually put issue numbers to track in whatsnew, the 39930 is not issue but pr, then users who check it might get confused.

Reply:
no they are the same (issue or pr), but yeah you can list all 3 (we usually list just the issues, however if the PR adds somehing non-trivial then can list that 2)